src/bullet.py

```diff
@@ -3,11 +3,21 @@
 from constants import SCREEN_HEIGHT, SCREEN_WIDTH
 from iron_math import add_vec2, rotate_vec2
 from player_manager import PlayerManager
+from typing import List, cast
+from arena.wall import SpriteForWall
 
 # For naming purposes, a bullet can be anything that comes out of a weapon included beams, rockets, etc
```

Comment:
Is this comment still true?  I think we discussed updating the name "bullet" here in a previous meeting, but I don't know the card that had the output of the discussion.

Reply:
I added this comment. I had started using the convention `weapon_output` for anything that comes out of a weapon. It felt really word and also confusing as `weapon_output_behavior` sounds like it's going to output a behavior. just using the term bullet to encompass projectiles, beams, mines etc is much cleaner if less accurate. The comment is meant to make this true for us.